lib/gringotts/gateways/pin_payments.ex

```diff
@@ -204,6 +240,18 @@ defmodule Gringotts.Gateways.PinPayments do
     |> respond
   end
 
+  defp commit_short(method, url, opts) do
```

Comment:
Instead of a new function `commit_short` you should pattern match on the `method`, and keep the same function name: `commit`